lib/spoom/cli/commands/coverage.rb

```diff
@@ -103,6 +103,47 @@ def timeline
           Spoom::Git.checkout(sha_before, path: path)
         end
 
+        desc "report", "produce a typing coverage report"
+        option :data, type: :string, desc: "Snapshots JSON data", default: DATA_DIR
+        option :file, type: :string, default: "spoom_report.html", aliases: :f
+        def report
+          in_sorbet_project!
+
+          data_dir = options[:data]
+          files = Dir.glob("#{data_dir}/*.json")
+          if files.empty?
+            message_no_data(data_dir)
+            exit(1)
+          end
+
+          snapshots = files.sort.map do |file|
+            json = File.read(file)
+            Spoom::Coverage::Snapshot.from_json(json)
+          end.filter(&:commit_timestamp).sort_by!(&:commit_timestamp)
```

Comment:
We actually have items/commits without a timestamp here? 😲

Reply:
Spoom allows you to take a snapshot of a project that doesn't uses Git so no commit sha nor timestamp (and also no timeline).